README.md

```diff
@@ -58,6 +58,43 @@ Or, to run from a local copy,
 }
 ```
 
+### Configuration with Local LLM and Goose
+
+For a more privacy preserving AI todo-list experience, you can run a local LLM using [LM Studio](https://lmstudio.ai/) or [Ollama](https://ollama.com/), and then point [Codename Goose](https://block.github.io/goose/) to use that LLM, and then configure it to run this MCP as a tool.
+
+```bash
+goose configure
+```
+
+The `configure` command will generate a config file which looks similar to the below configuration.
```

Comment:
The text makes it sound to me like the `goose configure` command automatically adds the todoist entry to the config, but I don't see how goose would know about the todoist MCP yet. Is it really that there are two steps here, first run `goose config`, which generates a barebones config, and then add the todoist entry to it?